src/AppInstallerCLICore/Workflows/MSStoreInstallerHandler.cpp

```diff
@@ -143,6 +153,17 @@ namespace AppInstaller::CLI::Workflow
 
         if (Manifest::ConvertToScopeEnum(context.Args.GetArg(Execution::Args::Type::InstallScope)) == Manifest::ScopeEnum::Machine)
         {
+            // TODO: There was a bug in InstallService where admin user is incorrectly identified as not admin,
+            // causing false access denied on many OS versions.
+            // Remove this check when the OS bug is fixed and back ported.
+            if (!Runtime::IsRunningAsSystem())
+            {
+                context.Reporter.Error() << Resource::String::InstallFlowReturnCodeSystemNotSupported << std::endl;

```

Comment:
That string is "The current system configuration does not support the installation of this package." If I saw that I would think the package is not supported at all, but I would like to know that I may at least be able to install it without admin privileges.

Reply:
The only applicable case where this will succeed is the InProcCom with system context. And for InProcCom, reporter strings are not reported. This is here mainly for consistency and for local system context testing. This will be cherry picked as hot fix for our second release candidate and we'd prefer not have a loc update to get it delayed, so if we are not happy with this string, I can just remove this line.

Reply:
How about `PendingWorkError`?

Reply:
I'm okay with keeping it as-is